inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -82,32 +83,49 @@ def contact_details(self, key, value):
     }
 
 
-@experiments.over('name_variants', '^419..')
-@utils.for_each_value
-def name_variants(self, key, value):
-    """Variants of the name."""
-    return value.get('a')
-
-
 @experiments.over('description', '^520..')
 @utils.for_each_value
 def description(self, key, value):
     """Description of experiment."""
     return value.get("a")
 
 
-@experiments.over('spokesperson', '^702..')
+@experiments.over('spokepersons', '^702..')
 @utils.for_each_value
-def spokesperson(self, key, value):
-    """Spokesperson of experiment."""
-    return value.get("a")
+def spokespersons(self, key, value):
+    """Spokepersons of the experiment."""
+    def _get_record(i_values):
+        i_value = force_single_element(i_values)
+        if i_value:
+            try:
+                recid = int(i_value.split('-')[1])
```

Comment:
It looks like it: https://inspirehep.net/search?ln=en&cc=Experiments&ln=en&cc=Experiments&p=702__i%3A%2F.*%2F+and+not+702__i%3A%2F%5EINSPIRE-%5B0-9%5D%2B%24%2F&action_search=Search&sf=experimentname&so=a&rm=&rg=25&sc=0&of=hb.

Shall I be extra safe and guard for the `IndexError`?